conmon-rs/server/src/rpc.rs

```diff
@@ -16,4 +18,26 @@ impl conmon::Server for Server {
         results.get().init_response().set_version(VERSION);
         Promise::ok(())
     }
+
+    fn create_container(
+        &mut self,
+        params: conmon::CreateContainerParams,
+        mut results: conmon::CreateContainerResults,
+    ) -> Promise<(), capnp::Error> {
+        let req = pry!(pry!(params.get()).get_request());
+        info!(
+            "Got a create container request for id {}",
+            pry!(req.get_id())
+        );
+        let args = self.generate_runtime_args(&params).unwrap();
+        let mut child = Command::new(self.config().runtime()).args(args).spawn().unwrap();
+        let pid = child.id().unwrap();
+        let id = req.get_id().unwrap().to_string();
+        tokio::spawn(async move {
```

Comment:
We will eventually want to send this pid over to a loop monitoring pid fds. (Not blocking for now.)